src/gui/guiFormSpecMenu.cpp

```diff
@@ -2431,10 +2424,21 @@ bool GUIFormSpecMenu::parseStyle(parserData *data, const std::string &element, b
 		spec.set(prop, value);
 	}
 
-	if (style_type) {
-		theme_by_type[selector] |= spec;
-	} else {
-		theme_by_name[selector] |= spec;
+	std::vector<std::string> selectors = split(parts[0], ',');
+	for (size_t sel = 0; sel < selectors.size(); sel++) {
+		std::string selector = trim(selectors[sel]);
+
+		if (selector.empty()) {
+			errorstream << "Invalid style element (Selector required): '" << element
```

Comment:
Might want to change the error message here to something like "Empty selector" since the context has changed